content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -152,14 +152,29 @@ The following table shows which referential action each database supports.
 
 | Database    | Cascade | Restrict | NoAction | SetNull | SetDefault |
 | :---------- | :------ | :------- | :------- | :------ | :--------- |
-| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
+| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**⌘ | **✔️**     |
 | MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**†    |
 | SQLite      | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
 | SQL Server  | **✔️**  | **❌**‡  | **✔️**   | **✔️**  | **✔️**     |
 | CockroachDB | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
 | MongoDB††   | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **❌**     |
 
-- † On MySQL, `SetDefault` is ignored and is silently replaced with `NoAction` instead.
+<Admonition type="warning">
+
+Although referential actions are an ANSI SQL feature, some relational databases diverged from the standard, giving rise to a few special cases.
+
+</Admonition>
+
+### Special cases for referential actions
+
+- † On MySQL (using the InnoDB storage engine), `SetDefault` is not supported, with differences depending on the database version. On MySQL 8+, `SetDefault` is considered an alias for `NoAction`.
+  One can define tables using the `SET DEFAULT` referential action, but once that action is triggered at runtime, a foreign key constraint error is raised.
+  On MySQL 5.6+, however, attempting to create one such table definition with the `SET DEFAULT` referential action fails with a syntax error.
+  MariaDB also lacks proper support for `SetDefault`.
+  For this reason, when setting `mysql` as the database provider, `prisma` warns users to replace `SetDefault` referential actions appearing in the Prisma schema with another action, to avoid undesired surprises at runtime.
+- ⌘ Postgres is the only database supported by Prisma that allows defining a `SetNull` referential action referring to a non-nullable field. However, once that action is triggered at runtime, a foreign key constraint error is raised.
+  For this reason, when setting `postgres` as the database provider in `foreignKeys` relation mode, `prisma` warns users to mark fields included in a `@relation` attribute with a `SetNull` referential action as optional.
+  For all other database providers, `prisma` would reject the schema with an explanatory validation error.
```

Comment:
I think this might be the intended information structure:
```suggestion
- † On MySQL (using the InnoDB storage engine), `SetDefault` is not supported, with differences depending on the database version. 
  - On MySQL 8+, `SetDefault` is considered an alias for `NoAction`. One can define tables using the `SET DEFAULT` referential action, but once that action is triggered at runtime, a foreign key constraint error is raised.
  - On MySQL 5.6+, however, attempting to create one such table definition with the `SET DEFAULT` referential action fails with a syntax error. 
  - MariaDB also lacks proper support for `SetDefault`.
  - For this reason, when setting `mysql` as the database provider, `prisma` warns users to replace `SetDefault` referential actions appearing in the Prisma schema with another action, to avoid undesired surprises at runtime.
- ⌘ Postgres is the only database supported by Prisma that allows defining a `SetNull` referential action referring to a non-nullable field. However, once that action is triggered at runtime, a foreign key constraint error is raised.
  - For this reason, when setting `postgres` as the database provider in `foreignKeys` relation mode, `prisma` warns users to mark fields included in a `@relation` attribute with a `SetNull` referential action as optional.
  - For all other database providers, `prisma` would reject the schema with an explanatory validation error.
```

Reply:
Looks good

Reply:
<img width="419" alt="image" src="https://user-images.githubusercontent.com/183673/204225692-aab1785a-dd1d-48ac-86dd-fadc5fbfba31.png">